free-1.md

```diff
@@ -6,63 +6,92 @@ good way to meet their needs.
 
 
 ### What's the need?
-_What need is met by your idea? Who are you helping? What is that person's
-experience like now? What would their experience be like if you could help 
-them?_
+Currently, it is difficult for a non-technical user to create a personal
+website with enough flexibility. Also, it is difficult for such a user to 
+understand the concepts behind how websites work such as the front-end and
+back-end. The DSL would be typed in Microsoft Word or Powerpoint and be able to 
+hook in and Excel spreadsheet essentially creating a dynamic website.
 
 
 ### Why a language?
-_Why is a DSL appropriate for your user(s)? How does it address the need?_
+I think a DSL would be a good fit for this because Word documents are
+similar to web framework templates which use a DSL. A DSL would allow
+users to work in a familiar environment (MS Office) with just a few changes
```

Comment:
I really like the idea of "familiar environment" and I think this would help people. I'm not sure exactly what you had in mind though; were you thinking of having people use modified Word macros or...?